boards/arc/emsdp/platform.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/init.h>
+
+#define DFSS_SPI0_BASE			0x80010000
+#define DFSS_SPI1_BASE			0x80010100
+#define DW_SPI_REG_CLK_ENA		(0x16)  /* DFSS only */
```

Comment:
If it's not related to the DW SPI IP, maybe don't hint on that with the definition name `DW_SPI_xxx`?